makefile

```diff
@@ -27,11 +31,7 @@ lint: format style typing
 
 .PHONY:update
 update:
-	pip-compile pyproject.toml --quiet --upgrade --resolver=backtracking --extra=dev --output-file=requirements/development.txt --unsafe-package django
-
-.PHONY:package
-package:
-	pip wheel .
+	uv pip compile pyproject.toml --quiet --upgrade --resolver=backtracking --extra=dev --output-file=requirements/development.txt --unsafe-package django
```

Comment:
Curious, why haven't we got django linked as a dependency in our prproject deps?

Reply:
I developed an aversion to adding Django to requirements long ago, but I must admit that I can't quite remember why now.

In this particular case I made sure to exclude it because it makes testing against different versions of Django easier with Tox.

Thankfully there's no reason to install this project without Django, so I can't imagine it causing an issue.

Reply:
This situation is fixed and clarified in https://github.com/kraken-tech/django-integrity/pull/30